### DISCUSSION TITLE
delve auto-complete

### DISCUSSION THREAD

**Discussion by guo-sj** [go-delve/delve#2943] [Q&A] [answered]
Hi, When I examine values of some Long-name variables, I have to type their full name by hand, which is tiresome. I wonder if the delve debugger supports auto-complete mechanism just like `TAB` key does in GDB. 

For example, for variable `shouldResize`, 
```
(dlv)  p should<TAB> => p shouldResize
```

Any help would be appreciated.

**Comment by derekparker** [comment, marked as answer]
Delve does support tab completion on certain symbols but not currently on local variables and such. We could add this as an enhancement.

**Reply by guo-sj** [reply]
Got it.

**Comment by gargnitingoogle** [comment]
+1 to the request.

**Reply by dennypenta** [reply]
just was fighting with dap-cmp extension and realized the debugger itself doesn't support it. If I can assist somehow to make it possible Im up
